src/pages/workspaces/workspace/Data.js

```diff
@@ -509,7 +510,7 @@ class EntitiesContent extends Component {
       }),
       showIgvSelector && h(IGVFileSelector, {
         onDismiss: () => this.setState({ showIgvSelector: false }),
-        onSuccess: selectedFiles => this.setState({ showIgvSelector: false, igvFiles: selectedFiles }),
+        onSuccess: (selectedFiles, refGenome) => this.setState({ showIgvSelector: false, igvFiles: selectedFiles, igvRefGenome: refGenome }),
```

Comment:
Instead of positional args, this might be better if it took/destructured from an object arg.